pkg/server/server.go

```diff
@@ -19,21 +19,29 @@ import (
 	"github.com/open-sauced/pizza/oven/pkg/providers"
 )
 
+// Config provides the configuration set on server startup
+// - Never Evict Repos: Repos that are preserved in cache regardless of LRU policy
+type Config struct {
+	NeverEvictRepos []string `yaml:"never-evict-repos"`
```

Comment:
In the actual config, this might make more sense as a hashmap for quick lookups during potential eviction vs having to iterate the whole list when checking if a repo should be evicted.

Reply:
Yeah, this makes sense! I partitioned the config step into a configParser and the actual config on the recent change. Otherwise, yaml for repos would have to look like this something like this which is not intuitive why the right side matters.

```yaml
never-evict-repos:
- https://github.com/kubernetes/kubernetes : 0
- https://github.com/open-sauced/pizza : 0
- https://github.com/open-sauced/pizza-cli : 0
```

The new config parser struct can parse it normally, and then designate it to the actual config drilled into the LRU cache (the map struct for O(1) lookup).

```go
	var configParser struct {
		NeverEvictRepos []string `yaml:"never-evict-repos"`
	}
```